ui/src/app/applications/components/application-create-panel/application-create-panel.tsx

```diff
@@ -170,6 +170,16 @@ export const ApplicationCreatePanel = (props: {
         }
     }
 
+    const onCreateApp = (data: models.Application) => {
```

Comment:
I see logic above from lines 138 to 171 that seems to address this issue. Can you provide an explanation on why these lines are not working correctly?

Reply:
The logic from these lines doesn't cover this problem - I've added steps to reproduce. You can test it, and you will see the problem is still there.